### DISCUSSION TITLE
GUI is not opening

### DISCUSSION THREAD

**Discussion by mercury-5** [The-OpenROAD-Project/OpenROAD-flow-scripts#1659] [Q&A] [answered]
make: *** No rule to make target 'gui_6_final.odb', needed by 'gui_final'.  Stop.

**Comment by maliberty** [comment]
You haven't got a 6_final.odb to open.  Something has previously not completed.

**Reply by mercury-5** [reply, marked as answer]
Thanks buddy! I added gui_final at the end and it worked

make DESIGN_CONFIG=./designs/sky130hd/vending_machine/config.mk gui_final
